alf/environments/carla_controller.py

```diff
@@ -78,29 +78,40 @@ class VehicleController(object):
     def __init__(self,
                  vehicle,
                  step_time,
-                 max_speed=20.,
+                 max_speed=5.56,
                  max_throttle=0.75,
                  max_steering=0.8,
                  max_brake=0.3,
-                 s_P=1.0,
-                 s_I=0.05,
+                 s_P=0.28,
+                 s_I=0.014,
                  s_D=0,
                  d_P=1.95,
                  d_I=0.07,
                  d_D=0.2):
         """
-        The defaults are from https://github.com/carla-simulator/carla/blob/master/PythonAPI/carla/agents/navigation/local_planner.py
+        The defaults are from https://github.com/carla-simulator/carla/blob/master/PythonAPI/carla/agents/navigation/local_planner.py.
+        Note that the max_speed and gain parameters for speed are originally
+        specified for speed in the unit of km/h. Since here we use m/s, we have
+        converted them as follows as our default values:
+            max_speed = 20 / 3.6 = 5.56
+            s_P = 1.0 / 3.6 = 0.28
```

Comment:
I feel should be "* 3.6". Suppose error is 1km/h, which is 1/3.6 m/s. We want to get the same control value, PID coefficient should be multiplied by 3.6.

Reply:
You're right. I initially used ``*3.6`` but later changed to ``/3.6`` due to some erroneous thoughts.
The unit free perspective is very helpful for resolving my confusing thoughts. 
I have corrected it.